packages/aws-cdk/bin/cdk.ts

```diff
@@ -281,8 +281,8 @@ async function initCommandLine() {
       success(' ⏳  Bootstrapping environment %s...', colors.blue(environment.name));
       try {
         const result = await bootstrapEnvironment(environment, aws, toolkitStackName, roleArn);
-        const message = result.noOp ? ' ✅  Environment %s was already fully bootstrapped!'
-                      : ' ✅  Successfully bootstrapped environment %s!';
+        const message = result.noOp ? ' ✅  Environment %s already bootstrapped, nothing to do.'
```

Comment:
would tend to shift the "nothing to do" upfront (it's the main info, I guess at this point).

```
Nothing to do: environment %s was already bootstrapped.
```

You might want to also apply the same polish to the deployment messaging (they are based off a very similar template).